db/academyPortal.sql

```diff
@@ -505,7 +505,18 @@ VALUES
 	(13,'Graduation',5,'Bath','2022-05-06','17:00:00','19:00:00',NULL,0),
 	(14,'Graduation',5,'Sheffield','2022-05-06','17:00:00','19:00:00',NULL,0),
 	(15,'Graduation',5,'Bath','2022-06-24','17:00:00','19:00:00',NULL,0),
-	(16,'Graduation',5,'Sheffield','2022-06-24','17:00:00','19:00:00',NULL,0);
+	(16,'Graduation',5,'Sheffield','2022-06-24','17:00:00','19:00:00',NULL,0),
+    (17,'Sprint Review',2,'Bath','2023-10-27','14:00:00','15:00:00','Toast Sandwich',0),
+    (18,'Hiring Event',1,'Bath','2023-10-28','14:00:00','15:00:00','Jam Sandwich',0),
+    (19,'Other',6,'Bath','2023-11-14','10:00:00','12:00:00',NULL,0),
+    (20,'Graduation',5,'Bath','2023-11-15','14:00:00','15:00:00',NULL,0),
+    (21,'Taster Session',3,'Remote','2023-11-20','14:00:00','15:00:00',NULL,0),
+    (22,'Other',6,'Remote','2023-11-21','19:00:00','22:00:00',NULL,0),
+    (23,'Assessment Day',4,'Remote','2023-11-24','14:00:00','15:00:00',NULL,0),
+    (24,'Sprint Review',2,'Bath','2023-12-10','09:00:00','12:30:00',NULL,0),
+    (25,'Graduation',5,'Bath','2023-12-15','15:00:00','18:00:00','Congratulations!!',0),
+    (26,'Taster Session',3,'Bath','2023-12-25','17:00:00','20:00:00','Chilli family noodles',0),
+    (27,'Other',6,'Bath','2023-12-29','09:00:00','17:00:00',NULL,0);
```

Comment:
can we have some more? use mockaroo to generate atleast 1 event every month for the next few years, ideally multiple